contracts/multisig/src/contract.rs

```diff
@@ -52,17 +52,14 @@ pub fn execute(
             sig_verifier,
         } => {
             execute::require_authorized_caller(&deps, info.sender)?;
-
-            let _sig_verifier = sig_verifier
-                .map(|addr| deps.api.addr_validate(&addr))
-                .transpose()?; // TODO: handle callback
```

Comment:
I'd actually prefer to keep this validation logic here actually, and pass an `Addr` into `start_signing_session`